### PR TITLE
feat: update intro and lecture-1 slides

### PR DESCRIPTION
**Pull request by artem-galas**
**Changes:**

- updated yarn to latest 3.1.1 version
- updated necessary lock file for `lecture-1`
- added some assets for slides
- copied some assets from prev year slides
- make different style for slides
- slightly update slides for Into and Lecture 1
  - added info about us/companies
  - added section about history of JS
 

**Testing:**
- go to `slides-2022` folder
- install deps `yarn`
- run remark `yarn serve -s intro.md` or `-s lecture-1.md`

### CODE REVIEW

File: slides-2022/intro.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+### Artem Halas
+- Software engineer in Pipedrive
+- Love cycling
+
+.full-image[![Artem Halas](assets/artem_halas.png)]
+]
+
+.fifty[
+### Stanislav Deviatykh
+- Software engineer in Qminder
```
Comment (artem-galas) [inline slides-2022/intro.md:48]: @stdevi please add info about yourself and add a picture

File: slides-2022/intro.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+# About Pipedrive
+- First CRM platform made for Salespeople
+- Estonian #5 Unicorn
+- 10 Offices in 8 countries
+- 900 + employees
+- Used by 95 000 Companies
+
+--
+# About Qminder
+
```
Comment (artem-galas) [inline slides-2022/intro.md:63]: @stdevi please add info about your company

**Comment by urmastalimaa** [comment]
Note that when merging to master, the content in GitHub pages is automatically updated via https://github.com/urmastalimaa/interactive-frontend-development/blob/master/.github/workflows/node.js.yml#L25

**Review by urmastalimaa — COMMENTED** [review]
A few comments:
* I think it is best to use the same slides and not partition them by year, this makes maintenance and reviewing easier. If one wants to see slides for 2011, there is always version control history. I.e. I would replace the `intro.md` in slides with slides-2022.
* Squash the commits into one, all three commits are concerned with lecture 1, no need for the intermediate states to be visible in final commit history.

**Comment by urmastalimaa** [comment]
I did look into ways to "import" a "about-presenters" slide, but unfortunately there are no good tools to do this. I think it is OK if you remove "about **me**" from slides and replace it with your bios, everything remains in history.

File: slides-2022/intro.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+* What cannot be understood, cannot be graded
+
+---
+
+# Homework
+
+* We've been reading and reviewing code for a long time
+* We will be reading and reviewing homework from each one of you
+* It is easy for me to detect when code was pieced together from multiple sources
+* I've had to deal with **plagiarism** **every year**
```
Comment (stdevi) [inline slides-2022/intro.md:160]: What do you think about changing to "We've had to deal" or maybe removing this line?

File: slides-2022/intro.md (hunk 2 above)
Comment (stdevi) [reply slides-2022/intro.md]: Done ✅

File: slides-2022/intro.md (hunk 1 above)
Comment (stdevi) [reply slides-2022/intro.md:48]: Done ✅

File: slides/intro.md [hunk 4]
```diff
@@ -1,41 +1,25 @@
...
-Presenter display slide notes
-
-Pre-lecture checklist
-
-* Switching between mirroring and non-mirroring works
-* Unrelated tabs and windows closed or minimized
-* Do not disturb mode **ON**
-* Slides cloned, one has presenters mode on
-* Dependencies installed and scripts are working for lecture example code
-* Terminal(s) open with code and window to run scripts
```
Comment (stdevi) [inline slides/intro.md:23]: It's a good idea to still have a pre-lecture checklist

File: slides/intro.md [hunk 5]
```diff
@@ -89,7 +77,7 @@ Direct all complaints concerning grades to her
 * UI components using React
 * Means of composition
 * Testing, testing, testing
---
+  --
```
Comment (stdevi) [inline slides/intro.md:80]: This slide looks a bit odd now:

image

Let's split it into 2?

**Review by stdevi — APPROVED** [review]
LGTM ✅

File: slides/js_ecosystem.md [hunk 6]
```diff
@@ -505,23 +504,22 @@ You can use additional tools, provided they are installed with `yarn install`, u
...
-# Homework result
+.full-image[![JS-tooling live reload](assets/lecture-1/hw-01-gif.gif)]
 
-<video width="100%" height="50%" controls>
-  <source src="assets/homework-mathemagician-1-result.m4v" type="video/mp4">
-  Your browser does not support the video tag.
-</video> 
+---
+class: middle, center
+#Questions?
```
Comment (urmastalimaa) [inline slides/js_ecosystem.md:525]: Minor note:
Generally with slides, "Questions?"/"Thanks for listening" end-slides are avoided. Here, similarly, it is best to "end" on the homework result GIF, which is going to be the most likely source of questions.

**Review by urmastalimaa — APPROVED** [review]

**Pull request merged**